client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	'Product reviews': i18n.translate( 'Product reviews' ),
+	'Product sales': i18n.translate( 'Product sales' ),
+	'Professional cleaning service': i18n.translate( 'Professional cleaning service' ),
+	'Professional portfolio': i18n.translate( 'Professional portfolio' ),
+	'Professional profile': i18n.translate( 'Professional profile' ),
+	'Professional services': i18n.translate( 'Professional services' ),
+	'Promotional marketing': i18n.translate( 'Promotional marketing' ),
+	'Promotional products': i18n.translate( 'Promotional products' ),
+	Promotions: i18n.translate( 'Promotions' ),
+	Property: i18n.translate( 'Property' ),
+	Psychotherapy: i18n.translate( 'Psychotherapy' ),
+	Quiz: i18n.translate( 'Quiz' ),
+	Quotes: i18n.translate( 'Quotes' ),
+	Quran: i18n.translate( 'Quran' ),
+	RVs: i18n.translate( 'RVs' ),
+	Racism: i18n.translate( 'Racism' ),
+	'Real estate': i18n.translate( 'Real estate' ),
+	Recipes: i18n.translate( 'Recipes' ),
+	'Recreation center': i18n.translate( 'Recreation center' ),
+	Recruiting: i18n.translate( 'Recruiting' ),
+	Recruitment: i18n.translate( 'Recruitment' ),
+	'Recruitment consultant': i18n.translate( 'Recruitment consultant' ),
+	Relationships: i18n.translate( 'Relationships' ),
+	Religion: i18n.translate( 'Religion' ),
+	'Renewable energy': i18n.translate( 'Renewable energy' ),
+	Renovation: i18n.translate( 'Renovation' ),
+	Reporter: i18n.translate( 'Reporter' ),
+	Research: i18n.translate( 'Research' ),
+	Researcher: i18n.translate( 'Researcher' ),
+	Resort: i18n.translate( 'Resort' ),
+	Resources: i18n.translate( 'Resources' ),
+	Restaurant: i18n.translate( 'Restaurant' ),
+	Resume: i18n.translate( 'Resume' ),
+	Retail: i18n.translate( 'Retail' ),
+	Reviews: i18n.translate( 'Reviews' ),
+	Robotics: i18n.translate( 'Robotics' ),
+	Romance: i18n.translate( 'Romance' ),
+	Running: i18n.translate( 'Running' ),
+	Sailing: i18n.translate( 'Sailing' ),
+	Sales: i18n.translate( 'Sales' ),
+	Salon: i18n.translate( 'Salon' ),
+	'Saving money': i18n.translate( 'Saving money' ),
+	Savings: i18n.translate( 'Savings' ),
+	Scholarship: i18n.translate( 'Scholarship' ),
+	School: i18n.translate( 'School' ),
+	'School activities': i18n.translate( 'School activities' ),
+	'School project': i18n.translate( 'School project' ),
+	'School website': i18n.translate( 'School website' ),
+	Science: i18n.translate( 'Science' ),
+	Scouts: i18n.translate( 'Scouts' ),
+	Security: i18n.translate( 'Security' ),
+	'Self cae': i18n.translate( 'Self cae' ),
+	'Self help': i18n.translate( 'Self help' ),
+	'Self improvement': i18n.translate( 'Self improvement' ),
+	'Self improvement services': i18n.translate( 'Self improvement services' ),
+	'Self-help': i18n.translate( 'Self-help' ),
+	'Selling art': i18n.translate( 'Selling art' ),
+	'Selling clothes': i18n.translate( 'Selling clothes' ),
+	'Selling products': i18n.translate( 'Selling products' ),
+	Seniors: i18n.translate( 'Seniors' ),
+	'Seo services': i18n.translate( 'Seo services' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **15** times:
`translate( 'Services' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).